src/segments/git.go

```diff
@@ -93,12 +93,14 @@ const (
 	MergeIcon properties.Property = "merge_icon"
 	// UpstreamIcons allows to add custom upstream icons
 	UpstreamIcons properties.Property = "upstream_icons"
-	// GithubIcon shows√ when upstream is github
+	// GithubIcon shows when upstream is github
 	GithubIcon properties.Property = "github_icon"
 	// BitbucketIcon shows  when upstream is bitbucket
 	BitbucketIcon properties.Property = "bitbucket_icon"
 	// AzureDevOpsIcon shows  when upstream is azure devops
 	AzureDevOpsIcon properties.Property = "azure_devops_icon"
+	// AwsCodecommitIcon shows  when upstream is aws codecommit
+	AwsCodecommitIcon properties.Property = "aws_codecommit_icon"
```

Comment:
```suggestion
	CodecommitIcon properties.Property = "codecommit_icon"
```

I feel this has a better ring to it, the remote prefix is also `codecommit` it seems.